controllers/apply-next/simple/form-model.js

```diff
@@ -376,6 +376,13 @@ module.exports = {
             }
         ],
         schema: schema,
-        processor: processor
+        processor: processor,
+        // @TODO these links will change when these docs are updated - should we just link to the programme page?
```

Comment:
Yes please, we should encourage web pages over PDFs where we can, so let's link to pages for now.